messaging/src/main/java/org/axonframework/queryhandling/SimpleQueryUpdateEmitter.java

```diff
@@ -276,6 +296,11 @@ private boolean inStartedPhaseOfUnitOfWork() {
         return Collections.unmodifiableSet(updateHandlers.keySet());
     }
 
+    @Override
+    public void closeActiveSubscriptions() {
```

Comment:
The introduction of this method warrants a test in the `SimpleQueryUpdateEmitterTest` class.